app/views/course/assessment/answer/programming/_annotation_topic.html.slim

```diff
@@ -1,2 +1,3 @@
-= render partial: 'course/discussion/topic', object: annotation_topic,
-         locals: { post_partial: 'course/assessment/answer/programming/annotation_post' }
+= div_for(annotation_topic)
```

Comment:
Referring to the above comment, we can do a check here `- unless annotation_topic.posts.empty?`